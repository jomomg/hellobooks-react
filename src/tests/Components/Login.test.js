import React from "react";
import Login from "../../components/login";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";

describe("<Login/>", ()=>{
    const location = {
        state: "/"
    };
    test("it renders successfully", ()=>{
        const login = shallow(<Login location={location}/>);
        expect(shallowToJson(login)).toMatchSnapshot();
    });
});