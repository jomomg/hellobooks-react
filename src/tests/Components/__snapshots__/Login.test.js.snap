// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login/> it renders successfully 1`] = `
<WithStyles(Paper)
  style={
    Object {
      "height": 280,
      "margin": "5% auto 5% auto",
      "width": 400,
    }
  }
>
  <WithStyles(Typography)
    component="h2"
    style={
      Object {
        "paddingTop": "3%",
        "textAlign": "center",
      }
    }
    variant="headline"
  >
    Login to Hello Books
  </WithStyles(Typography)>
  <form
    noValidate={true}
    onSubmit={[Function]}
    style={
      Object {
        "display": "flex",
        "flexWrap": "wrap",
      }
    }
  >
    <TextField
      id="email"
      label="Email"
      margin="normal"
      onChange={[Function]}
      placeholder="Your email address"
      required={false}
      select={false}
      style={
        Object {
          "marginLeft": "4%",
          "marginRight": "4%",
          "width": "400px",
        }
      }
      type="email"
      value=""
    />
    <TextField
      id="password"
      label="Password"
      margin="normal"
      onChange={[Function]}
      placeholder="Enter a password"
      required={false}
      select={false}
      style={
        Object {
          "marginLeft": "4%",
          "marginRight": "4%",
          "width": "400px",
        }
      }
      type="password"
      value=""
    />
    <WithStyles(Button)
      color="primary"
      style={
        Object {
          "backgroundColor": "orange",
          "color": "black",
          "margin": "4% auto 0% auto",
        }
      }
      type="submit"
      variant="extendedFab"
    >
      LOGIN
    </WithStyles(Button)>
  </form>
  <Notifier />
</WithStyles(Paper)>
`;
